### PR TITLE
Fixed #4723 and added about windows compilation

### LINKED ISSUE #4723: AppVeyor build url returns error
The [AppVeyor build url](https://ci.appveyor.com/project/vigsterkr/shogun) in the [Windows section](https://github.com/shogun-toolbox/shogun/blob/develop/doc/readme/INSTALL.md#manual-windows) of the INSTALL docs return an error.

The url is currently: https://ci.appveyor.com/project/vigsterkr/shogun

Screen Shot 2019-07-17 at 2 21 49 AM

### PR DESCRIPTION
**Pull request by krutikabapat**
Fixed Issue #4723 and added about windows compilation.

### CODE REVIEW

File: doc/readme/INSTALL.md [hunk 1]
```diff
@@ -286,7 +286,7 @@ On a Linux cluster without root access, using [Anaconda](https://www.continuum.i
 
 ## Windows build <a name="manual-windows"></a>
 
-Please see our [AppVeyor](https://ci.appveyor.com/project/vigsterkr/shogun) build.
+Please see our [Azzure Pipelines](https://dev.azure.com/shogunml/shogun/_build?definitionId=2) build.
```
Comment (gf712) [inline doc/readme/INSTALL.md]: `Azure`

Comment (krutikabapat) [reply doc/readme/INSTALL.md]: @gf712 I have fixed the typo!. Please review and le me know if anything has to be changed further

**Review by iglesias — APPROVED** [review]

File: doc/readme/INSTALL.md [hunk 2]
```diff
@@ -286,7 +286,7 @@ On a Linux cluster without root access, using [Anaconda](https://www.continuum.i
 
 ## Windows build <a name="manual-windows"></a>
 
-Please see our [AppVeyor](https://ci.appveyor.com/project/vigsterkr/shogun) build.
+Please see our [Azure Pipelines](https://dev.azure.com/shogunml/shogun/_build?definitionId=2) build.
 It is recommended to use "Visual Studio 14 2015" or "MSBuild".
```
Comment (vigsterkr) [inline doc/readme/INSTALL.md]: actually Visual Studio 16 2019

File: doc/readme/INSTALL.md [hunk 3]
```diff
@@ -297,7 +297,13 @@ You will need to adjust all path names to the Windows style, e.g.
 
 You need to specify a different generator in cmake (to match your IDE), e.g.
 
-    cmake -G"Visual Studio 14 2015 Win64" -DCMAKE_BUILD_TYPE=Release -DBUILD_META_EXAMPLES=OFF -DENABLE_TESTING=ON ..
+    cmake -G"Visual Studio 14 2015 Win64" -DCMAKE_BUILD_TYPE=Release -DBUILD_META_EXAMPLES=OFF -DENABLE_TESTING=ON ..
```
Comment (vigsterkr) [inline doc/readme/INSTALL.md]: see above
```
cmake -G"Visual Studio 16 2019" -DCMAKE_BUILD_TYPE=Release -DBUILD_META_EXAMPLES=OFF -DENABLE_TESTING=ON ..  
```

File: doc/readme/INSTALL.md [hunk 4]
```diff
@@ -297,7 +297,13 @@ You will need to adjust all path names to the Windows style, e.g.
 
 You need to specify a different generator in cmake (to match your IDE), e.g.
 
-    cmake -G"Visual Studio 14 2015 Win64" -DCMAKE_BUILD_TYPE=Release -DBUILD_META_EXAMPLES=OFF -DENABLE_TESTING=ON ..
+    cmake -G"Visual Studio 14 2015 Win64" -DCMAKE_BUILD_TYPE=Release -DBUILD_META_EXAMPLES=OFF -DENABLE_TESTING=ON ..  
+The above ```cmake``` has the following arguments:    
+1. ```Visual Studio 14 2015 Win64``` specifies the target platform to be x64.
```
Comment (vigsterkr) [inline doc/readme/INSTALL.md]: change it to Visual Studio 16 2019

File: doc/readme/INSTALL.md [hunk 5]
```diff
@@ -297,7 +297,13 @@ You will need to adjust all path names to the Windows style, e.g.
 
 You need to specify a different generator in cmake (to match your IDE), e.g.
 
-    cmake -G"Visual Studio 14 2015 Win64" -DCMAKE_BUILD_TYPE=Release -DBUILD_META_EXAMPLES=OFF -DENABLE_TESTING=ON ..
+    cmake -G"Visual Studio 14 2015 Win64" -DCMAKE_BUILD_TYPE=Release -DBUILD_META_EXAMPLES=OFF -DENABLE_TESTING=ON ..  
+The above ```cmake``` has the following arguments:    
+1. ```Visual Studio 14 2015 Win64``` specifies the target platform to be x64.   
+2. ```DCMAKE_BUILD_TYPE=Release``` specifies a build type and asks compiler to perform optimization and omit debug information.
```
Comment (vigsterkr) [inline doc/readme/INSTALL.md]: -DCMAKE_BUILD_TYPE=Release

**Review by vigsterkr — CHANGES_REQUESTED** [review]

File: doc/readme/INSTALL.md (hunk 5 above)
Comment (krutikabapat) [reply doc/readme/INSTALL.md]: @vigsterkr Thank you for all the suggestions and reviews. I have made the suggested changes. Let me know if any other changes are required.

**Pull request merged**